src/allmydata/test/test_connections.py

```diff
@@ -11,7 +11,7 @@
 from ..util.tor_provider import create as create_tor_provider
 
 
-BASECONFIG = ""
+BASECONFIG = b""
```

Comment:
Similarly, if `config_from_string` can accept either unicode or bytes, you won't have to make all these changes.